packages/playground/src/pages/Dropin/Dropin.js

```diff
@@ -9,13 +9,17 @@ import '../../style.scss';
 const initCheckout = async () => {
     const paymentMethodsResponse = await getPaymentMethods({ amount, shopperLocale });
 
+    //    paymentMethodsResponse.paymentMethods[0].brands.push('elo');
```

Comment:
Can we clean up the changes on this file?

Reply:
ok

Reply:
ok